src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,127 @@
+package duke;
+
+import duke.exception.*;
+
+/**
+ * Parses and contains the important info required
+ * for each call to Duke methods given through CLI.
+ */
+public class Parser {
+    public String comparator;
+    private String commandLine;
+    private String commandWord;
+    private String taskName;
+    private String taskDate;
+    private Integer taskNumber;
+
+    Parser() {
+        comparator = "";
+        commandLine = "";
+        commandWord = "";
+        taskName = "";
+        taskDate = "";
+        taskNumber = 0;
+    }
+
+    /**
+     * Sets the command line of <code>Parser</code> to
+     * a given string inputted via CLI
```

Comment:
Same as above regarding adding a full stop.

Reply:
Got it, thanks!